index.go

```diff
@@ -442,6 +442,13 @@ func main() {
 	})
 
 	m.Group("/webProperties", func(r martini.Router) {
+		// TODO /requirement routes do not appear to be used. Remove them if not used by Nov 6, 2017
+		// verified by grepping all project code, stackdriver logs, and google analytics
+		//r.Post("/requirement/:id", webProperty_controller.CreateUpdateWebPropertyRequirement) // TODO remove route/controller if not used by Nov 2, 2017 //updates when an id is present; otherwise, creates
+		//r.Put("/requirement", webProperty_controller.CreateUpdateWebPropertyRequirement)      // TODO remove route/controller if not used by Nov 2, 2017 //updates when an id is present; otherwise, creates
+		//r.Delete("/requirement/:id", webProperty_controller.DeleteWebPropertyRequirement)     // TODO remove route/controller if not used by Nov 2, 2017 //{id}
+		//r.Get("/requirement/:id", webProperty_controller.GetWebPropertyRequirement)           // TODO remove route/controller if not used by Nov 2, 2017 //{id}
+		//r.Get("/requirement", webProperty_controller.GetAllRequirements)                      // TODO remove route/controller if not used by Nov 2, 2017
```

Comment:
That api lists customer website information. You can view this in the admin.

I didn't see anywhere for requirements though. In the database it looks like they are just notes like
> Customer service phone number is prominently displayed on website.

I didn't see anywhere in the admin that would list stuff like that. (or in code anywhere looking for the /requirement route to query for them)